src/module/data/effect/base.mjs

```diff
@@ -6,12 +6,49 @@ export default class BaseEffectModel extends foundry.abstract.TypeDataModel {
     type: "base"
   });
 
+  /** @override */
+  static LOCALIZATION_PREFIXES = ["DRAW_STEEL.Effect.base"];
+
+  /** @override */
   static defineSchema() {
     const fields = foundry.data.fields;
     const config = ds.CONFIG;
     return {
-      end: new fields.StringField({choices: config.effectEnds, blank: false}),
-      characteristic: new fields.StringField({choices: config.characteristics, blank: false})
+      end: new fields.StringField({choices: Object.keys(config.effectEnds), blank: true, required: true})
```

Comment:
Do we expect this to remain simple enough that a mere StringField is warranted? How about further nesting:
```js
end: {
  type: "...",
}
```
which - I could imagine - may have further properties in the future, e.g., to determine conditions of ending the effect.

Reply:
At least in the current rules the conditions are one of the three choices, BUT should maybe include a roll modifier field for the Saving Throw type?